helmfile/upstream/cert-manager/values.yaml

```diff
@@ -442,6 +498,23 @@ cainjector:
     # labels: {}
     automountServiceAccountToken: true
 
+  # Automounting API credentials for a particular pod
+  # automountServiceAccountToken: true
+
+acmesolver:
```

Comment:
seems that they introduced a new way to use a custom acmesolver image, so the [old instructions](https://github.com/elastisys/compliantkubernetes-apps/blob/main/config/config/wc-config.yaml#L208:L213) may not apply any more

please check and expose the new values in our configs, a migration script will also be required

Reply:
So they have introduced this new way, but the old way through extraArgs should still work and take precedence if set, see [this PR](https://github.com/cert-manager/cert-manager/pull/5693). Should these changes still be made?

Reply:
I am ok with not making this changes